app/modules/sightings/models.py

```diff
@@ -719,9 +719,9 @@ def send_identification(
 
         # Message construction has to be in the task as the jobId must be unique
         job_uuid = uuid.uuid4()
-        matching_set_query = matching_set_query_override or self.id_configs[
-            config_id
-        ].get('matching_set')
+        matching_set_query = matching_set_query or self.id_configs[config_id].get(
```

Comment:
not really concerned about this one, I can tweak it later but what I meant was that  matching_set_query should be a mandatory, not optional parameter in this function and in the send_identification task, then it was always set to the correct value in the call to send_identification (which it is)